dr_martens/checkitem.py

```diff
@@ -0,0 +1,22 @@
+import jwt
+import datetime
+
+from django.http import JsonResponse
+from django.conf import settings
+
+
+class CheckItem:
+
+    def check_keys_in_body(check_dict:dict, necessary_key_list:list):
+
+        check_dict_keys = check_dict.keys()
+    
+        if not set(necessary_key_list).issubset(set(check_dict_keys)):
+            required_keys = set(necessary_key_list) \
+                          - set(necessary_key_list).intersection(set(check_dict_keys))
+
+            error_message='' 
+            for required_key in required_keys:
+                error_message += required_key + ", "
+
+            raise KeyError(f"[{error_message}] was not found in http body")
```

Comment:
```suggestion
from django.http import JsonResponse
from django.conf import settings

class CheckItem:
    def check_keys_in_body(check_dict:dict, necessary_key_list:list):
        check_dict_keys = check_dict.keys()
    
        if not set(necessary_key_list).issubset(set(check_dict_keys)):
            required_keys = set(necessary_key_list) \
                          - set(necessary_key_list).intersection(set(check_dict_keys))

            error_message = '' 
            
            for required_key in required_keys:
                error_message += required_key + ", "

            raise KeyError(f"[{error_message}] was not found in http body")
```
wecode 컨벤션만 맞추시면 될 것 같습니다.

Reply:
리뷰 감사합니다.
컨벤션에 맞춰서 수정하였습니다.